src/client/app/components/ui/Like.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Like = ({count, likeAdd, postId}) => (
-  <div>
-    <button className='like' onClick={() => likeAdd(postId)}>
-      Like {count}
-    </button>
-  </div>
-);
+class Like extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { count, postId, likeAdd } = this.props;
+    return (
+      this.renderLike(count, postId, likeAdd)
```

Comment:
4:1  warning  Component should be written as a pure function  react/prefer-stateless-function
4:1  warning  Component should be written as a pure function  react/prefer-stateless-function